bin/kn

```diff
@@ -44,6 +44,7 @@ cloud providers CLI:
 developers:
   bash                   run an interactive bash promt in the provisioners container
   kubetoken              generate and print a kubeadm token
+  git
```

Comment:
- Add git to help output in kn

Reply:
Perhaps add a short description for git: `run git`. Did you encounter a case in which the used didn't have it in his system?

Reply:
Occationally good to have when working with openshift from deployer node....since it is under developer section I guess we could remove it without notice in future if it is unneccessary